pkg/splunk/enterprise/configuration.go

```diff
@@ -630,15 +630,24 @@ func updateSplunkPodTemplateWithConfig(ctx context.Context, client splcommon.Con
 			Value: spec.LicenseURL,
 		})
 	}
```

Comment:
During the time we support both CRDs it's possible a CR has both `LicenseMasterRef` and `LicenseManagerRef`. In that case, we default to `LicenseManagerRef` to support the migration.